src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.cs

```diff
@@ -176,7 +193,7 @@ private Glyph GetGlyph(SymbolKind kind, Accessibility declaredAccessibility)
                     originalDefinition == valueTaskType ||
                     originalDefinition.SpecialType == SpecialType.System_Nullable_T)
```

Comment:
❓ Is `Nullable<int?>` a situation we need to worry about? It's not semantically correct but it is expressible in source code.

Reply:
Will add a test to see.

Reply:
Test now added.